src/libs/LangChain.Core/Chains/StackableChains/Agents/GroupChat.cs

```diff
@@ -18,6 +18,8 @@ public class GroupChat : BaseStackableChain
     private readonly string _outputKey;
     
     int _currentAgentId;
+    private readonly MessageFormatter _messageFormatter;
+    private readonly ChatMessageHistory _chatMessageHistory;
     private readonly ConversationBufferMemory _conversationBufferMemory;
```

Comment:
If I understand correctly, we can remove this

Reply:
Do you mean that I should remove this file completely?

Reply:
I mean conversation history line

Reply:
Oh, right!  Sorry, missed that unused reference.